composer.json

```diff
@@ -3,11 +3,13 @@
     "type": "ezplatform-bundle",
     "require": {
         "php": "~5.4 || ~7.0",
-        "ezsystems/ezpublish-kernel": "~5.3.5@dev || ~5.4.2@dev || ^6.0@dev",
+        "ezsystems/ezpublish-kernel": "~5.3.5@dev || ~5.4.2@dev || ~6.11@dev",
         "fzaninotto/faker": "~1.4",
         "jdorn/sql-formatter": "~1.2"
     },
-
+    "require-dev": {
+        "phpunit/phpunit": "^4.8.35"
```

Comment:
Let's try to use PHPUnit 6 since we don't have existing tests (right? ;) ), 
but also let's make sure that this doesn't create conflict when required from meta-repo.